### PR TITLE
[HY-726] First take at adding EU location detection

### PR DESCRIPTION
**Pull request by LBRapid**
This adds a helper `request_from_eu?` to any app including `actionpack-cloudfront`. It just looks at a regex of country codes and compares that to the `Cloudfront-Viewer-Country` header.

If this looks like an okay approach, I will add tests and Rollbar logging (when the header is empty).

Also I don't know a ton about the ip_ranges work already being done here and whether or not that code being active for all apps in production environment would be an issue.

### CODE REVIEW

File: lib/action_pack/cloudfront/eu_location_detection.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+module ActionPack
+  module Cloudfront
+    module EuLocationDetection
+      EU_ISO_3166_ALPHA_2 = /^(AL|AD|AT|BY|BE|BA|BG|HR|CY|CZ|DK|EE|FI
+                              |FR|DE|GI|GR|HU|IS|IE|IM|IT|RS|LV|LI|LT
+                              |LU|MK|MT|MD|MC|ME|NL|NO|PL|PT|RO|RU|SM
+                              |RS|SK|SI|ES|SE|CH|UA|GB|VA|RS)$/ix
+
+      def request_from_eu?
+        country_code = ActionDispatch::Http::Headers.new(request.headers)['CloudFront-Viewer-Country']
```
Comment (metaskills) [inline lib/action_pack/cloudfront/eu_location_detection.rb]: Mind explains why `request.headers['CloudFront-Viewer-Country']` would not work?

Comment (LBRapid) [reply lib/action_pack/cloudfront/eu_location_detection.rb]: It does! I'm not sure, maybe I was setting the header incorrectly when I first started or something and ended up with that after some troubleshooting. But now I'm moving back to the simpler way since it works as well.
